[IMP] accounting/l10n_in: update india localization doc

eapa-odoo opened the pull request:
Revise the India localization documentation to ensure it aligns with the updated
functionality in the master branch.

ref: https://github.com/odoo/odoo/pull/190265

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/12058.png)](https://mergebot.odoo.com/odoo/documentation/pull/12058)

eapa-odoo commented:
Hello @afma-odoo,

Thank you for reviewing my PR and for your suggestions.

I have implemented the requested changes, and since the e-Invoice feature has been updated, I've adjusted that as well.

Could you please review it again?

Thanks! 😊

afma-odoo commented on content/applications/finance/fiscal_localizations/india.rst:
```diff
@@ -332,30 +310,23 @@ You can print the invoice PDF report once you have submitted the E-Way bill. The
+:guilabel:`Cancel e-Waybill`. In the :guilabel:` Cancel Ewaybill` window, fill out the
+:guilabel:`Cancel Reason` and :guilabel:`Cancel Remarks`.
 
 .. image:: india/e-waybill-cancellation.png
    :alt: Cancel reason and remarks
```
Could you resize the screenshot to only keep the "Cancel Ewaybill" window?

afma-odoo reviewed (approved):
Hello @eapa-odoo,
Thank you for the update. I approved it, could you just resize the screenshot in the "E-way bill cancellation" section? Thanks a lot! 🙂

eapa-odoo commented:
Hello, @afma-odoo 
I have updated the screenshot, can you please have a look?

Thanks :grinning:

afma-odoo commented:
Hi @eapa-odoo! 
Thank you, everything's fine now 😉

auva-odoo reviewed (approved):
@robodoo r+
